### PR TITLE
chore: update app translations

### PR DESCRIPTION
**Pull request by firebomb1**
## Description

Weekly translation flow by the Globalization Workstream.

As mentioned on Discord, after the Monorail update we've decided to removed the English copies from the asset description override translations files from `asset-service` that came from the `lib` repo.

The more updated English copies from the English file will be used automatically when a translation is missing. A couple of actual (but outdated) translations are also removed in the process (in languages that are not active anyways).

## Pull Request Type

- [ ] :bug: Bug fix (Non-breaking Change: Fixes an issue)
- [x] :hammer_and_wrench: Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)
- [ ] :nail_care: New Feature (Breaking/Non-breaking Change)

### CODE REVIEW

File: src/assets/translations/es/main.json [hunk 1]
```diff
@@ -1910,5 +1926,22 @@
         "cta": "Crea una"
       }
     }
+  },
+  "nft": {
+    "floorPrice": "Precio Base ",
```
Comment (gomesalexandre) [inline src/assets/translations/es/main.json:1931]:
```suggestion
    "floorPrice": "Precio Base",
```

File: src/assets/translations/es/main.json [hunk 2]
```diff
@@ -1910,5 +1926,22 @@
...
+    "lastPrice": "Ultimo Precio",
+    "overview": "Descripción",
```
Comment (gomesalexandre) [inline src/assets/translations/es/main.json:1933]: ?

File: src/assets/translations/pt/main.json [hunk 3]
```diff
@@ -1910,5 +1926,22 @@
         "cta": "Crie um"
       }
     }
+  },
+  "nft": {
+    "floorPrice": "Floor Price",
```
Comment (gomesalexandre) [inline src/assets/translations/pt/main.json:1931]: Forgotten translation?

Comment (firebomb1) [reply src/assets/translations/pt/main.json:1931]: Portuguese translator deliberately left as is, based on his experience.

File: src/assets/translations/es/main.json (hunk 2 above)
Comment (firebomb1) [reply src/assets/translations/es/main.json:1933]:
```suggestion
    "overview": "Visión general",
```

**Comment by firebomb1** [comment]
Thanks for the review @gomesalexandre ! The Spanish translation was edited after giving more context to our translator.

**Review by gomesalexandre — APPROVED** [review]
LGTM now 🥮

**Pull request merged**